src/config/feature-flags.ts

```diff
@@ -41,8 +41,8 @@ const getLaunchDarklyValue = async (flag: BooleanFlags | StringFlags, defaultVal
 		await launchdarklyClient.waitForInitialization();
 		const user = createLaunchdarklyUser(jiraHost);
 		return launchdarklyClient.variation(flag, user, defaultValue);
-	} catch (err: unknown) {
-		logger.error(`Error resolving value for feature flag "${flag}"`, err);
+	} catch (err) {
+		logger.error({flag, ...err}, "Error resolving value for feature flag");
```

Comment:
no, it's just an object.

Reply:
Not sure if it is the best idea to add a feature flag as a log param. Hence I don't think it will be a common case to look for the feature flag parameter in Splunk. And having it in params instead of message will make it less visible